docs/UserGuide.adoc

```diff
@@ -233,14 +233,15 @@ Shows a list of all of the specified `TYPE`. +
 Format: `list TYPE`.
 
 ****
-* `TYPE` can be of the following: `contact`, `student`, `task`, `appointment`.
+* `TYPE` can be of the following: `contact`, `student`, `task`, `appointment`, `shortcut`.
 * `TYPE` cannot be empty.
```

Comment:
The change you made to the list command now makes type plural, so it should be the following: "contacts", "students", "tasks", "shortcuts", appointments"

Be careful that your command is up to date with the UserGuide, we have UserTesting today!